EdgeOfEmpireBot/EdgeOfEmpireBot/Models/Models.cs

```diff
@@ -0,0 +1,10 @@
+namespace EdgeOfEmpireBot.Models;
+
+public class Game
```

Comment:
Do we need to specify SteamGame, in the event we have game model for tabletop stuff (like sprawl, masks etc)

Reply:
I agree this should definitely be more specific. Even the name "Game" isn't correct it should be "SteamGameDetails"